src/player.ts

```diff
@@ -38,6 +38,95 @@ export interface PlayerConfig {
    * ```
    */
   licenseKey?: string;
+  /**
+   * Configures playback behavior. A default `PlaybackConfig` is set initially.
```

Comment:
Actually, we don't set a default value for `PlaybackConfig` so this doc comment could be a little misleading.

```suggestion
   * Configuration options for player's default playback behavior.
```